app/_locales/en/messages.json

```diff
@@ -862,6 +865,12 @@
   "message": {
     "message": "Message"
   },
+  "metaMaskConnectStatusParagraphOne": {
+    "message": "This is the new MetaMask Connect status indicator. From here you can easily see and manage sites you’ve connected to with your MetaMask wallet."
+  },
+  "metaMaskConnectStatusParagraphTwo": {
+    "message": "Click the connect status to see your connected sites and their permissions."
```

Comment:
Nit: We say "Connect status" in the other message, but "connect status" here. We should probably be consistent about whether or not we're capitalizing this.

Reply:
Done in cd231a311